### PR TITLE
worked on task 6 and added relevant tests, created new comments files…

### PR DESCRIPTION
**Pull request by krahma93**
… for controller and model

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -1,116 +1,150 @@
...
-            .then(({ body }) => {
-                expect(body.msg).toBe("Article id does not exist")
-            }) 
-    })
-    test("Status 400 -  when submitting an invalid ID", () => {
+          .get("/api/articles/1/comments")
+          .expect(200)
+          .then(({ body: { comments } }) => {
+            expect(comments.length).toBe(11);
+            expect(comments[0]).toEqual(
```
Comment (Dutton1990) [inline __tests__/app.test.js]: Instead of just checking the first comment, use a forEach again to loop over and check every single one

**Review by Dutton1990 — CHANGES_REQUESTED** [review]
There's another test you can add, what if an article exists but has no comments? 

This should return a 200 because it's a correct request, but also return an empty array of comments. 

See if you can add this test and figure out how you could go about this. You might need to check that the article exists 🤔 

Send this PR again please once you've done that. Nice work otherwise!

**Review by leekli — CHANGES_REQUESTED** [review]
Hi Khaled, thanks for making the changes Christian asked.

There's 1 more request, which looks like you had it before but you took it out?

You need a test and some logic to deal with trying to get comments for an article which does not exist. For example if I went to /api/articles/768787/comments... that article_id does not exist. Have another look at getting this implemented.

Once complete, please submit another pull request to us for review.

**Review by leekli — APPROVED** [review]
Fantastic, well done Khaled!

You are free to merge this branch :D

**Pull request merged**